db/schema.rb

```diff
@@ -50,4 +47,5 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  add_foreign_key "cars", "users"
```

Comment:
Not sure why it has been added nor why application worked previously without this line